main.go

```diff
@@ -422,7 +422,18 @@ func run() (err error) {
 	return
 }
 
+func handlePanic() {
+	// detect if panic occurred or not
+	a := recover()
+	if a != nil {
+		logger.Info("Panic: ", a)
+		os.Exit(1)
```

Comment:
What happens if we don't keep os.Exit(1)?

Reply:
Nothing, but I thought exit status 1 would show an error, as I found it in many places. It Fixed.